docs/business-apps/introduction-to-sharepoint-business-process-integration.md

```diff
@@ -8,39 +8,39 @@ localization_priority: Priority
+Office 365 provides connections to a range of third-party systems with Power Automate and Power Apps. Power Automate lets you easily automate business processes while Power Apps lets you create exceptional forms and mobile apps based on SharePoint files and data. Both Power Automate and Power Apps feature intuitive visual designers, so anyone can build forms and screens in an easy-to-use interface.
 
-## Microsoft Flow
+## Power Automate
 
-Using Microsoft Flow, you can create workflows through an easy-to-use visual designer that will guide you through each step and help authenticate your accounts.
+Using Power Automate, you can create workflows through an easy-to-use visual designer that will guide you through each step and help authenticate your accounts.
 
-Select from dozens of pre-made templates also available in Flow to start automating your app’s usage. Here are just a few applications of these templates:
+Select from dozens of pre-made templates to start automating your app’s usage. Here are just a few applications of these templates:
 
 * Create a new record in Dynamics CRM when a new list item is added to a SharePoint list.
 * Copy new members from MailChimp lists to SharePoint lists.
 * Move files to different folders after they’re approved in SharePoint.
 * Create a new item in SharePoint when a new order is added in Salesforce.
-* Route finished documents to a team for approval
+* Route finished documents to a team for approval.
 
-Whether you start with a template or begin from scratch, using Flow to create automation features is intuitive and easy to understand.
+Whether you start with a template or begin from scratch, using Power Automate to create automation features is intuitive and easy to understand.
 
-Cross-tool functionality seamlessly moves your data from one operation to another. Since SharePoint is a core piece of the Office 365 suite,your data easily integrates with tools like Excel or Power BI reporting. And since it uses an Excelinspired expression language, these tools talk with each other, using formula language you already recognize and understand to integrate your data from one function to the next.
+Cross-tool functionality seamlessly moves your data from one operation to another. With SharePoint being a core component of the Microsoft 365 suite, your data easily integrates with tools like Excel or Power BI reporting. And because it uses an Excel-inspired expression language, these tools talk with each other, using formula language you already recognize and understand to integrate your data from one function to the next.
 
-## Microsoft PowerApps
+## Power Apps
```

Comment:
If we change the title as suggested earlier, we could change the title of this section to "Building Business Apps".

Reply:
Change made. If approved, merge #5528 into master from branch jackwi-alt